template.spec

```diff
@@ -1,32 +1,26 @@
 %global pkgname {{ pypi['info']['name'] }}
 
-Name:           python3-%{pkgname}
+# todo: This is not valid specfile
```

Comment:
Why? And how does a comment like that is useful?

Reply:
forget to delete note.